src/slack_api_util.ts

```diff
@@ -198,7 +210,11 @@ export async function fetchSlackUserName(
     return response.data.user.real_name;
   } else {
     logger.error(
-      `SLACKAPIUTIL.fetchSlackUserName: Failed to reveal Slack user name (${userId}). Error: ${response.data.error}.`
+      `SLACKAPIUTIL.fetchSlackUserName: Failed to reveal Slack user name (${userId}). Error: response.data: ${JSON.stringify(
+        response.data
```

Comment:
Is stringifying both the `data` and `response_metadata` necessary? Isn't `response_metadata` already included in the data?

Reply:
Good catch thanks. Stupid mistake. Should be fixed now.